Bir şehirde yaşayan insanların %20si yalnız yaşamaktadır. Yalnız yaşayan insanların %60ı yalnız yaşamayanların ise %20si evcil hayvan beslemektedir. Buna göre evcil hayvanı olduğu bilinen birinin yalnız yaşama olasılığı nedir? 

In [3]:
yalniz_yasayan_insan = 0.2
yalniz_yasamayan_insan = 0.8

yalniz_evcil_hayvan_sahibi = yalniz_yasayan_insan*0.6
yalniz_olmayan_evcil_hayvan_sahibi = yalniz_yasamayan_insan*0.2

evcil_hayvan_sahibi = yalniz_evcil_hayvan_sahibi + yalniz_olmayan_evcil_hayvan_sahibi

ornek_uzay = evcil_hayvan_sahibi
olay_uzayi = yalniz_evcil_hayvan_sahibi

print(olay_uzayi/ornek_uzay)

0.4285714285714285


Bayes teoremine göre;

P(A|B) = $\frac{P(A \cap B)}{P(B)}$

Yukarıdaki probleme uyarlayacak olursak;

P(B) = evcil hayvan sahibi olma olasılığı

P(B) = $0.2 \times 0.6$ + $0.8 \times 0.2$ = 0.28

$P(A \cap B)$ = Yalnız evcil hayvan sahipleri = $0.2 \times 0.6$ = 0.12

$\frac{P(A \cap B)}{P(B)}$ = $\frac{0.12}{0.28}$ = 0.4285714285714285